app-creator/client/src/widgets/attributes-tab/attributes-tab.ts

```diff
@@ -641,14 +643,49 @@ ${value}</textarea
     }
   }
 
+  private getDisabledStyles(type: string): Set<string> | null {
```

Comment:
I think it makes sense to encode the style parameters as string literal types so we can impose a stricter set of allowed parameters for `disabledStyles`. I think it would help guard against typos!

https://www.typescriptlang.org/docs/handbook/literal-types.html